exonum/src/events/noise/wrapper.rs

```diff
@@ -24,7 +25,9 @@ use events::noise::HandshakeParams;
 pub const NOISE_MAX_MESSAGE_LENGTH: usize = 65_535;
 pub const TAG_LENGTH: usize = 16;
```

Comment:
Off-topic: The code in `encrypt_msg` and `decrypt_msg` (which uses this constant, here is the link) is nearly incomprehensible to me. I think at the very least it should be augmented with asserts and comments, and tested.

Reply:
Agree, I will add a task for that.